lib/turbolinks/x_domain_blocker.rb

```diff
@@ -4,18 +4,18 @@ module Turbolinks
   # - The request is being redirected to a different domain
   module XDomainBlocker
     private
-      def same_origin?(a, b)
```

Comment:
Well, `same_origin?` has been there since [March 2013](https://github.com/rails/turbolinks/commit/f8fe25831b372f3ea2c0a62fed35dc101b1ae0a4) and I found zero issues on Rails mentioning `same_origin?` as breaking their applications.

I think your fear is a little overblown here and prefixing `_turbolinks_` reads like a crutch to me. But I'll leave it there :smile: